src/client/App.jsx

```diff
@@ -0,0 +1,105 @@
+import { useState, useEffect } from 'react';
+import { Routes, Route } from 'react-router-dom';
+
+import CreateProfile from './pages/register/CreateProfile';
+import CreateUser from './pages/register/CreateUser';
+import Footer from './Footer';
+import Forum from './pages/forum/Forum';
+import Home from './pages/home/Home';
+import Header from './Header';
+import Leaderboard from './pages/leaderboard/Leaderboard';
+import LeftMenu from './LeftMenu';
+import Login from './pages/login/Login';
+
+import { LOCAL_STORAGE, INT_LINK, USER_URL } from './config';
+
+import './styles/app.css';
+
+const App = () => {
+    const [isLoggedIn, setIsLoggedIn] = useState(false);
+    const [user, setUser] = useState(null);
+
+    console.log('state', {
```

Comment:
This looks like a debugging print statement - if so this should not be introduced into the main branch